crates/oxc_linter/src/snapshots/valid_params.snap

```diff
@@ -0,0 +1,146 @@
+---
+source: crates/oxc_linter/src/tester.rs
+---
+  ⚠ eslint-plugin-promise(valid-params): Promise.resolve() requires 0 or 1 arguments, but received 2
+   ╭─[valid_params.tsx:1:1]
+ 1 │ Promise.resolve(1, 2)
+   · ─────────────────────
+   ╰────
+
+  ⚠ eslint-plugin-promise(valid-params): Promise.resolve() requires 0 or 1 arguments, but received 5
+   ╭─[valid_params.tsx:1:1]
+ 1 │ Promise.resolve({}, function() {}, 1, 2, 3)
+   · ───────────────────────────────────────────
+   ╰────
+
+  ⚠ eslint-plugin-promise(valid-params): Promise.reject() requires 0 or 1 arguments, but received 3
+   ╭─[valid_params.tsx:1:1]
+ 1 │ Promise.reject(1, 2, 3)
+   · ───────────────────────
+   ╰────
+
+  ⚠ eslint-plugin-promise(valid-params): Promise.reject() requires 0 or 1 arguments, but received 4
+   ╭─[valid_params.tsx:1:1]
+ 1 │ Promise.reject({}, function() {}, 1, 2)
+   · ───────────────────────────────────────
+   ╰────
+
+  ⚠ eslint-plugin-promise(valid-params): Promise.race() requires 1 argument, but received 2
+   ╭─[valid_params.tsx:1:1]
+ 1 │ Promise.race(1, 2)
+   · ──────────────────
+   ╰────
+
+  ⚠ eslint-plugin-promise(valid-params): Promise.race() requires 1 argument, but received 5
+   ╭─[valid_params.tsx:1:1]
+ 1 │ Promise.race({}, function() {}, 1, 2, 3)
+   · ────────────────────────────────────────
+   ╰────
+
+  ⚠ eslint-plugin-promise(valid-params): Promise.all() requires 1 argument, but received 3
+   ╭─[valid_params.tsx:1:1]
+ 1 │ Promise.all(1, 2, 3)
+   · ────────────────────
+   ╰────
+
+  ⚠ eslint-plugin-promise(valid-params): Promise.all() requires 1 argument, but received 4
+   ╭─[valid_params.tsx:1:1]
+ 1 │ Promise.all({}, function() {}, 1, 2)
+   · ────────────────────────────────────
+   ╰────
+
+  ⚠ eslint-plugin-promise(valid-params): Promise.allSettled() requires 1 argument, but received 3
+   ╭─[valid_params.tsx:1:1]
+ 1 │ Promise.allSettled(1, 2, 3)
+   · ───────────────────────────
+   ╰────
+
+  ⚠ eslint-plugin-promise(valid-params): Promise.allSettled() requires 1 argument, but received 4
+   ╭─[valid_params.tsx:1:1]
+ 1 │ Promise.allSettled({}, function() {}, 1, 2)
+   · ───────────────────────────────────────────
+   ╰────
+
+  ⚠ eslint-plugin-promise(valid-params): Promise.any() requires 1 argument, but received 3
+   ╭─[valid_params.tsx:1:1]
+ 1 │ Promise.any(1, 2, 3)
```

Comment:
Could we add a suggestion for this case? Using `ctx.diagnostic_with_suggestion`?

Reply:
Hmm well the arguments can be any type so I haven't figured out how to get the span location yet of the first argument. Or use the source text to determine the location of the comma